### ISSUE TITLE
Pixelated button icons in oscilloscope view

### ISSUE THREAD

**Issue by mliberty1** [jetperch/pyjoulescope_ui#1]
Observed on Ubuntu 18.04.1 LTS:
![joulescope_2](https://user-images.githubusercontent.com/5393422/52644180-55c90200-2eac-11e9-9b1e-259da4616d01.png)

**Comment by mliberty1** [comment]
Note that joulescope_ui.main contains special high-DPI code for windows:

```
    if sys.platform.startswith('win'):
        ctypes.windll.user32.SetProcessDPIAware()
```

Is something like this needed for Ubuntu?

**Comment by mliberty1** [comment]
The solution was to use SVG images and stylesheets.  The QIcon approach is broken by default for high DPI, and I could not figure out how to make it cooperate.

**Comment by mliberty1** [comment]
Fixed in 0.9.1.
